### ISSUE TITLE
Unable to call service

### ISSUE THREAD

**Issue by pedolsky** [dummylabs/thewatchman#147]
Using service watchman.report causes an error. It doesn’t matter whether I use a pre-configurated notify service or the overwrite feature. However, the messages are sent.

Watchman version 0.6.3
HA version 2024.8.1 Supervised

Examples:

```
action: watchman.report
data:
  create_file: false
  send_notification: true

service: watchman.report
data:
  create_file: false
  send_notification: true
  service: notify.gmail
```

Logs:

```
Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.watchman
Quelle: custom_components/watchman/__init__.py:425
Integration: Watchman (Dokumentation, Probleme)
Erstmals aufgetreten: 11:30:05 (12 Vorkommnisse)
Zuletzt protokolliert: 12:05:11

Unable to call service notify.persistent_notification due to an error.
Unable to call service notify.gmail due to an error.
```

**Comment by dsz79** [comment]
Este error se originó a partir de una integración personalizada.

Registrador: custom_components.watchman
Fuente: custom_components/watchman/__init__.py:425
integración: Watchman (documentación, problemas)
Ocurrió por primera vez: 13:55:55 (1 ocurrencias)
Último inicio de sesión: 13:55:55

Unable to call service notify.mobile_app_pixel_7 due to an error.


The notification was send, but in the log show this error.

**Comment by dummylabs** [comment]
API for service call has been changed a while ago, this harmless error message should be fixed in 0.6.4
